generatePrefixesList.js

```diff
@@ -0,0 +1,427 @@
+iffdfmport axios from "axios";
+import url from "url";
+import brembo from "brembo";
+import merge from "deepmerge";
+import batchPromises from "batch-promises";
+import RpkiValidator from "rpki-validator";
+import { AS } from "./model";
+
+const apiTimeout = 120000;
+const clientId = "ntt-bgpalerter";
+const rpki = new RpkiValidator({clientId});
+import axiosEnrich from "./utils/axiosEnrich";
+
+import https from 'https';
+
+const httpsAgentOptions = {
+    keepAlive: (process.env.KEEP_ALIVE || 'true') === 'true',
```

Comment:
Would this always be true?

Reply:
It looks like it, but I'm just propagating changes from https://github.com/appneta/BGPalerter/commit/8707d2ea2fe51f4746da62ce80665ba672a2c1bb#diff-e943a4a381385ea088e7616d23007ce5652c485bf88a04e25bd1710826b92df6R10

so I am hesitant to make the change